webapp/app/config/config.yml

```diff
@@ -278,3 +278,10 @@ nelmio_api_doc:
                 items:
                     type: string
                     description: A single ID
+
+ssp_guard:
+    installation_path: /srv/simplesamlphp
```

Comment:
Do you think so? Seems more of the dbhost genre which are in this config rule. Not sure if we should put fs paths in the db.

Reply:
Well but this file is checked into git currently, while we *generate* the DB file (and ignore it), right?

Reply:
This to me sounds like it should come from `configure`, which is where you would specify non-standard external libraries. And then it would be substituted during `make build` time.